### ISSUE TITLE
Passing data while either app is inactive?

### ISSUE THREAD

**Issue by NorthMcCormick** [mutualmobile/MMWormhole#72]
Hey guys I'm using WatchOS 2 and iOS 9

My app has a timer that runs for a user defined amount of time, which can get pretty dangerous with background tasks and I'm worried about asking apple to approve an app that could potentially drain the users battery, which is why I use a local notifications instead of a background mode.

I'm now working in my watch app with WatchOS2 and I'm running into an updating issue where if I wake the watch and ping the phone app isn't responding back because it's asleep, and if the user doesn't have their watch app open while the iPhone app is sending it updates, it's not receiving them and updating the UI...

So here's what I'm wondering, is there anyway for maybe just the watchOS app to continue to get those updates from the iPhone even if it is not awake? It is using a workout session so it's always the forefront of the app. I'm just trying to make sure I'm not missing something before I attempt to dive into this a different way and possibly make more time.

Thanks! (2.0 has been splendid so far by the way amazing work!)

**Comment by NorthMcCormick** [comment]
After a couple more hours of research...

I did get this figured out and I'm wondering if someone would be able to modify the docs or if I did a PR if that is okay.

let wormholeSession = MMWormholeSession.sharedListeningSession()
    let wormhole = MMWormhole(applicationGroupIdentifier: "group.com.redfitapp", optionalDirectory: nil, transitingType: .SessionContext)

Live data only works in my apps if I use wormholeSession and listen with that. But the regular wormhole is the one that seems to handle the cached/queued messages that get pushed and read when WatchOS/iOS seems fit. 

Just something that I think would help a lot of developers who use this across watchos2.
